kms/attestations/verify_attestation.py

```diff
@@ -69,3 +70,18 @@ def verify(attestation_file, bundle_file):
                 continue
         return False
 # [END verify_attestations]
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+            description=__doc__)
+    parser.add_argument('attestation_file', help="Name of attestation file.")
+    parser.add_argument('bundle_file', help="Name of certificate bundle file.")
+
+    args = parser.parse_args()
+
+    if verify(args.attestation_file, args.bundle_file):
+        print('Signature verified.')
+    else:
+        print('Signature verification failed.')
+
```

Comment:
There are 2 newline characters at the end of the file.
Maybe the editor @bkuang is using can not add just one newline?

Anyways, I'll try to fix it.